test/resources/example-data.js

```diff
@@ -47,11 +48,12 @@ const EX_4_HTML_CONVERTER_OUTPUT = '<!DOCTYPE html><html lang="en"><head><title>
   '<p>This is an introduction to something cool.</p>\n' +
   '</body></html>';
 
-const EX_5_HTML_CONVERTER_HTML_INFO = {
-  engine: 'md',
-  file: path.resolve('test/resources/layout-in-layout-test/page.md'),
-  layoutsDir: path.resolve('test/resources/layout-in-layout-test/')
-};
+const EX_5_HTML_CONVERTER_HTML_INFO = new HTMLInfo(
```

Comment:
Why is this test data from another test updated?

Reply:
This test didn't create an actual instance of the HTMLInfo class causing calls to my added function to fail. Now that the function is static it no longer matters.